### DISCUSSION TITLE
Fooocus crashes every time I use Inpaint Mode and FaceSwap (Mixing Image Prompt and Inpaint enabled)

### DISCUSSION THREAD

**Discussion by AndreiLungu64** [lllyasviel/Fooocus#1318] [General]
Every time I use this feature and trying to make a faceswap the program stops. Are there any fixes?

[Fooocus] Encoding positive #1 ...
[Fooocus Model Management] Moving model(s) has taken 0.14 seconds
[Fooocus] Encoding negative #1 ...
[Fooocus] Image processing ...
Upscaling image with shape (673, 673, 3) ...
[Fooocus] VAE Inpaint encoding ...
^C

**Comment by Ramsello** [comment]
Hello Friend. I have the same problem. Perhaps you found a solution?

**Comment by Ramsello** [comment]
> Every time I use this feature and trying to make a faceswap the program stops. Are there any fixes?
> 
> [Fooocus] Encoding positive #1 ... [Fooocus Model Management] Moving model(s) has taken 0.14 seconds [Fooocus] Encoding negative #1 ... [Fooocus] Image processing ... Upscaling image with shape (673, 673, 3) ... [Fooocus] VAE Inpaint encoding ... ^C

Hello Friend. I have the same problem. Perhaps you found a solution?

**Comment by miraculix95** [comment]
The same ...  it does a ^C after this section

"Requested to load CLIPVisionModelWithProjection
Loading 1 new model"

what is your environment? Me, a Google CoLab environment with additional compute purchased

**Comment by akelaonline** [comment]
SAme problem here

**Comment by eddyizm** [comment]
If you guys are using google colab, pay close attention to your resources. You can see the graph and if it redlines, it will crash. That will give you an idea of what is maxing it out. Once you identify what is redlining you can address it a bit with some flags.
